src/main/java/org/apache/sling/i18n/impl/JcrResourceBundleProvider.java

```diff
@@ -167,6 +171,9 @@
 
     private BundleTracker<Set<LocatorPaths>> locatorPathsTracker;
     private List<LocatorPaths> locatorPaths = new CopyOnWriteArrayList<>();
+    
+    // Ignore events in these paths
```

Comment:
There is no need to copy the values around. Just access directly from the config object.